tools/debuggers.md

```diff
@@ -0,0 +1,86 @@
+# Debuggers
```

Comment:
Shouldn't we also be giving some practical guidance on how to debug? Or is this to be covered in online lectures or external videos? For instance:

- How and where to set breakpoints (what are they); how to clear them
- How to inspect variables and scope
- How to examine the call stack
- How to step over, step in, step out, resume
- etc etc.

Reply:
yup!  I was thinking of this file as just a starter to set things in a direction.  once there's something it's easier to contribute and adjust